src/convos/conversation-states.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: conversation-states
+---
+
+# Conversation States
+
+Throughout the life of the conversation, it can be in various states.
+These are represented by which tab of the [Conversation List](https://app.ab.bot/conversations) they appear under.
```

Comment:
This might be clearer as two sentences: "The Conversation List contains tabs for each of these states. Clicking the tab will list the conversations currently in that state."